config/05-ce-auth-defaults.conf

```
@@ -33,7 +33,7 @@ SCHEDD.ALLOW_WRITE = $(USERS), condor@daemon.htcondor.org/$(FULL_HOSTNAME)
 COLLECTOR.ALLOW_ADVERTISE_MASTER = $(FRIENDLY_DAEMONS)
 COLLECTOR.ALLOW_ADVERTISE_SCHEDD = $(FRIENDLY_DAEMONS)
 COLLECTOR.ALLOW_ADVERTISE_STARTD =  $(UNMAPPED_USERS), $(USERS)
-SCHEDD.ALLOW_NEGOTIATOR = condor@daemon.htcondor.org/$(FULL_HOSTNAME)
+SCHEDD.ALLOW_NEGOTIATOR = condor@$(UID_DOMAIN)
```

```suggestion
SCHEDD.ALLOW_NEGOTIATOR = condor@$daemon.htcondor.org
```

The `@daemon.htcondor.org/$(FULL_HOSTNAME)` may not be ideal as we've discussed (maybe there's a ticket for it?) but I don't think `UID_DOMAIN` would be right either due to:

- https://github.com/htcondor/htcondor-ce/blob/V6-branch/config/05-ce-auth-defaults.conf#L22-L22
- https://github.com/htcondor/htcondor-ce/blob/V6-branch/config/mapfiles.d/50-common-default.conf#L13